pkg/cri/server/container_create_windows.go

```diff
@@ -129,3 +131,22 @@ func (c *criService) containerSpec(
 func (c *criService) containerSpecOpts(config *runtime.ContainerConfig, imageConfig *imagespec.ImageConfig) ([]oci.SpecOpts, error) {
 	return nil, nil
 }
+
+// snapshotterOpts returns any Windows specific snapshotter options for the r/w layer
+func snapshotterOpts(snapshotterName string, config *runtime.ContainerConfig) []snapshots.Opt {
+	var opts []snapshots.Opt
+
+	switch snapshotterName {
```

Comment:
Isn't it possible to share this implementation across platforms? The function checks the snapshotter and the underlying platforms don't matter.

Reply:
I kept it this way in case we needed to do some operations that would only build on Windows/Unix. e.g. Windows needs to call some win32 API or same story for Unix. I suppose that may be forward thinking as that's not happening in this current PR. Let me know if you'd prefer to just have this be shared.